specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -306,35 +401,36 @@ http://contoso-app.azurewebsites.net/login
 @doc("The input artifacts for the test.")
 model TestInputArtifacts {
   @doc("File info")
-  configFileInfo?: FileInfo;
+  configFileInfo?: TestFileInfo;
 
   @doc("File info")
-  testScriptFileInfo?: FileInfo;
+  testScriptFileInfo?: TestFileInfo;
 
   @doc("File info")
-  userPropFileInfo?: FileInfo;
+  userPropFileInfo?: TestFileInfo;
 
   @doc("File info")
-  inputArtifactsZipFileInfo?: FileInfo;
+  inputArtifactsZipFileInfo?: TestFileInfo;
+
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("The config json file for url based test")
+  urlTestsConfigFileInfo?: TestFileInfo;
 
   @doc("Additional supported files for the test run")
   @visibility("read")
-  additionalFileInfo?: FileInfo[];
+  additionalFileInfo?: TestFileInfo[];
 }
 
 @doc("File info")
 model FileInfo {
   @doc("File URL.")
   url?: string;
```

Comment:
why is url, fileType, fileName all nullable?

Reply:
url can be null if validation fails. fileName is required in current implementation, will update the fileType and other field after confirmation from Krishna.

Reply:
they are marked as optional as per our GA spec.